### PR TITLE
feat(oui-navbar): update oui-navbar to new specs

### PR DESCRIPTION
**Pull request by AxelPeter**
UU-532

Staging: http://feature_oui_navbar.ui-kit.ovh (main navbar)

http://feature_oui_navbar.ui-kit.ovh/#!/ovh-ui-kit/navbar

### CODE REVIEW

File: packages/oui-navbar/_mixins.less [hunk 1]
```diff
@@ -6,136 +6,461 @@
...
+        color: @arrow-color;
+        padding: @padding;
+        position: absolute;
+        top: 50%;
+        right: 0;
+        transform: translateY(-50%);
+      }
+    }
+
+    .active > & { color: @font-color_active; }
```
Comment (neolitec) [inline packages/oui-navbar/_mixins.less]: Not BEM compliant.

File: packages/oui-navbar/_mixins.less [hunk 2]
```diff
@@ -6,136 +6,461 @@
...
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+    background: black;
+    transition: opacity 0.25s ease-out;
+    pointer-events: none;
+    z-index: @z-index;
+
+    @media screen and (max-width: 1024px) {
```
Comment (neolitec) [inline packages/oui-navbar/_mixins.less]: We have responsive mixins to handle this. If not, is it possible to redefine another mixins and therefore avoid redefining break-points?

File: packages/oui-navbar/_mixins.less [hunk 3]
```diff
@@ -6,136 +6,461 @@
...
+      max-width: @menu-width;
+      box-shadow: none;
     }
 
-    &__title-bold {
-      display: inline-block;
-      margin-left: rem-calc(8);
-      font-size: rem-calc(28);
-      font-weight: 600;
+    @media screen and (min-width: 1025px) {
```
Comment (neolitec) [inline packages/oui-navbar/_mixins.less]: I wonder if 1025 is an error or not...

File: packages/oui-navbar/_mixins.less [hunk 4]
```diff
@@ -6,136 +6,461 @@
...
-      font-weight: 600;
-      text-decoration: none;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      cursor: pointer;
     }
 
-    .@{select-selector} {
-      margin-bottom: 0;
+    &-icon {
```
Comment (neolitec) [inline packages/oui-navbar/_mixins.less]: (optional) Maybe just define that it defines the hamburger icon ;)

File: packages/oui-navbar/README.md [hunk 5]
```diff
@@ -14,31 +14,249 @@ oui-navbar is a package which provide styles for the navbar component.
 
 ```html:preview
```
Comment (neolitec) [inline packages/oui-navbar/README.md]: Change to html only (maybe temporarly)

File: packages/oui-navbar/_mixins.less [hunk 6]
```diff
@@ -6,136 +6,461 @@
...
+    flex-direction: @flex-direction;
+    flex-grow: @flex-grow;
+    align-items: @align-items;
+    justify-content: @justify-content;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    // List items
+    &-item {
```
Comment (jimmyfortinx) [inline packages/oui-navbar/_mixins.less]: We are using bem with `__` instead of `-`

File: packages/oui-navbar/_mixins.less (hunk 2 above)
Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: Yes, i'll look for them

File: packages/oui-navbar/_mixins.less (hunk 3 above)
Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: Not an error: min-width (for desktop only)

File: packages/oui-navbar/README.md (hunk 5 above)
Comment (AxelPeter) [reply packages/oui-navbar/README.md]: The doc will be changed with simpler examples :)

File: packages/oui-navbar/_mixins.less (hunk 6 above)
Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: I know, but it's for element like `list-item` (like `menu-item` too).
Or maybe put directly `&__menu-item`, and change it in `navbar.less` ?

File: packages/oui-navbar/_mixins.less (hunk 4 above)
Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: Yes :)

File: packages/oui-navbar/_mixins.less (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: Done

**Pull request merged**